src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js

```diff
@@ -74,6 +75,7 @@ const ActivityThreadReplies = ({
                         onDelete={onDelete}
                         onEdit={onEdit}
                         permissions={getReplyPermissions(reply)}
+                        allowCollapse
```

Comment:
Minor, but we typically try to keep our props, destructuring, etc. alphabetized when possible.